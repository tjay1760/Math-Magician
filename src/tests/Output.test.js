import { render } from '@testing-library/react';
import ScreenOutput from '../components/Output';

describe('ScreenOutput', () => {
  test('should render the ScreenOutput component correctly', () => {
    const calculatorState = {
      next: null,
      total: 0,
    };

    const { container } = render(<ScreenOutput calculatorState={calculatorState} />);

    expect(container).toMatchSnapshot();
  });

  // Add more test cases as needed
});
