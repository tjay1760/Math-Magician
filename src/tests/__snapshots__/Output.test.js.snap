// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScreenOutput should render the ScreenOutput component correctly 1`] = `
<div>
  <section
    id="output-screen"
  >
    <p>
       
      <output>
         
        0
      </output>
    </p>
  </section>
</div>
`;

exports[`Should render as expected test render of quote page 1`] = `
<div>
  <section
    id="output-screen"
  >
    <p>
       
      <output>
         
      </output>
    </p>
  </section>
</div>
`;
